pkgs/top-level/all-packages.nix

```diff
@@ -2199,7 +2199,11 @@ with pkgs;
 
   libretro = recurseIntoAttrs
     (callPackage ../applications/emulators/retroarch/cores.nix {
-      retroarch = retroarchBare;
+      retroarch = retroarchBare.override {
+        # Avoid core rebuilds in case of bumps in either assets or core info files
+        withAssets = false;
+        withCoreInfo = false;
+      };
```

Comment:
(I don't know a lot about retroarch or its Nix expression, so please excuse/ignore anything dumb I might say.)

What else is retroarchBare used for? I get the feeling that withAssets/withCoreInfo should default to false, to match upstream for retroarchBare and then have non-bare versions be wrapped with assets etc. as `retroarch` or `retroarch-full` attributes?

Reply:
I thought about this, but this would make everything much more confusing so for now this is the best solution. Will probably need another refactor to fix this in the proper way.

Reply:
Ok.

Reply:
Did what you suggested. It was not as confusing as I thought initially, so should be fine.